package.json

```diff
@@ -22,6 +22,8 @@
     "@stdlib/math-base-special-ldexp": "^0.0.5",
     "dlv": "^1.1.3",
     "dset": "^3.1.1",
+    "jsrsasign": "^10.8.6",
+    "node-rsa": "^1.1.1",
```

Comment:
I don't think you still need the node-rsa dep, correct?

Reply:
yes, will remove that